sympy/physics/optics/utils.py

```diff
@@ -422,3 +424,31 @@ def lens_formula(focal_length=None, u=None, v=None):
         if focal_length == oo:
             return Limit(u*_f/(u + _f), _f, oo).doit()
         return u*focal_length/(u + focal_length)
+
+def hyperfocal_distance(f = None, N = None, c = None):
```

Comment:
I've removed the defaults. Running the tests locally to see it works fine before committing.